### PR TITLE
Does my coding partner follow JS best practices?

### PR DESCRIPTION
**Pull request by joseph07-drack**
### Learning Objectives:
---

- Give feedback on someone else's project based on clean code best practices.

- Apply knowledge the student has gained in the past when giving a code review.

- Perform a simple code review for a coding partner.

- Understand existing code in a way that allows the student to give a good code review.

### Instructions:
---
- Once you get a pull request link from your partner
- Check the quality of the code (you can use previous lessons as a guide)
 [DRY, KISS, and YAGNI](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/clean-code/lesson_DRY_KISS_YAGNI.md) rules.
[HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).
- Finish your review with a nice summary.
- Compare your review with the issue created by the project's author in the [Do I follow JavaScript best practices?](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/clean-code/exercises/clean_code_self_assessment.md) exercise.

### CODE REVIEW

**Review by TahenyBELGUITH — CHANGES_REQUESTED** [review]
Hi @joseph07-drack , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+.tasks .task {
+  display: flex;
+  justify-content: space-between;
+  font-size: 15px;
+  margin-bottom: 10px;
+  border-bottom: 1px solid #4343;
+
+  /* border-width: 100; */
+  padding: 0 5px;
+}
```
[the remark this answers is not in the document]
Comment (joseph07-drack) [reply src/style.css:78]: Oh!!! I forgot to remove that unused line of code 😋😋. I am going to remove it right now.
Thank you for the review. 👍

**Review by TahenyBELGUITH — APPROVED** [review]

**Comment by TahenyBELGUITH** [comment]
Hi @joseph07-drack ,


## Wow, you did it 🎉
Brilliant
### Well done!




Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by joseph07-drack** [comment]
Thank you @TahenyBELGUITH for the review 👍

**Pull request merged**